exp-models/addon/serializers/account.js

```diff
@@ -7,15 +7,22 @@ let bcrypt = dcodeIO.bcrypt;
 
 export default DS.JSONAPISerializer.extend(JamSerializer, JamDocumentSerializer, {
     modelName: 'account',
-
     serialize(record, options) {
-      if(record.record.get('isNew')) {
-        record = record.record;
-        var salt = bcrypt.genSaltSync(12);
-        record.set('password', bcrypt.hashSync(record.get('password'), salt));
-        record = record._createSnapshot();
-      }
+        record.profiles = record._profiles;
+        delete record._profiles;
```

Comment:
See note above about renaming the computed. We may be able to simplify the serializer logic.